api/database/data/qa/03_lynda_snell_logs.sql

```diff
@@ -0,0 +1,116 @@
+----Adding a volunteer log----
```

Comment:
All logs are created by the volunteer they are associated with. It would be good to have some data added by an alternative `VOLUNTEER_ADMIN` for testing purposes

Reply:
Up to the QA to define the data they need for test cases they define. Doesn't seem like this is needed for the current user stories, can be added later.